docs/usage_stats_collection.md

```diff
@@ -0,0 +1,54 @@
+# Usage Stats Collection
+
+Skyplane collects usage stats by default. All data is anonymous and helps us maintain an open-source map of cloud network performance data. This data is used to improve the speed of your transfers by routing your traffic around slow links. It will only be used by the Skyplane team to improve the services and for research purposes.
```

Comment:
`Skyplane collects usage stats by default` => `Skyplane collects performance statistics. Some of this data is shared with the Skyplane developers to improve the accuracy of the Skyplane solver.`